Gemfile.lock

```diff
@@ -22,22 +22,22 @@ GEM
     ffi (1.9.18)
     forwardable-extended (2.6.0)
     gemoji (3.0.0)
-    github-pages (139)
+    github-pages (146)
```

Comment:
all those changes just to keep up to date or there is a specific new module/feature ?

I assume "rake test" still works then good!